packages/nodes-base/nodes/Grist/GenericFunctions.ts

```diff
@@ -72,10 +71,10 @@ export function parseSortProperties(sortProperties: GristSortProperties) {
 }
 
 export function parseFilterProperties(filterProperties: GristFilterProperties) {
- return filterProperties.reduce<{ [key: string]: string[]; }>((acc, cur) => {
+	return filterProperties.reduce<{ [key: string]: string[]; }>((acc, cur) => {
 		acc[cur.field] = acc[cur.field] ?? [];
-		const values = cur.values.split(',').map(v => v.trim());
```

Comment:
If the values cannot be separated by commas, then there needs to be an update to:

- The docs https://github.com/n8n-io/n8n-docs/pull/619/files#diff-57ee899626d8d1192a3ec966c6a285f41905ac1007ef4684082b400d2cd7c542R115
- The field name, displayName, and description https://github.com/n8n-io/n8n/pull/2158/files#diff-1e5990e55548d4729b196e41b6b3894516c358970965696360ff203c16977a74R161